platform/commonUI/edit/src/capabilities/EditorCapability.js

```diff
@@ -101,10 +101,15 @@ define(
          */
         EditorCapability.prototype.finish = function () {
             var domainObject = this.domainObject;
-            return this.transactionService.cancel().then(function () {
-                domainObject.getCapability("status").set("editing", false);
-                return domainObject;
-            });
+
+            if (this.transactionService.isActive()) {
+                return this.transactionService.cancel().then(function () {
+                    domainObject.getCapability("status").set("editing", false);
+                    return domainObject;
+                });
+            } else {
+                return new Promise(function () {});
```

Comment:
This promise doesn't ever resolve.  need to return a promise that will resolve; there's a shorthand way to do it...